src/DiffEq/MixDirichletCoeffPolicy.h

```diff
@@ -139,13 +139,13 @@ class MixDirichletHomCoeffConst {
       tk::real sumY = 0.0;
       for (ncomp_t c=0; c<ncomp; ++c) sumY += Y[c];
 
-      // Y|Kc
-      std::vector< tk::real > YK( ncomp, 0.0 );
-      for (ncomp_t c=0; c<ncomp; ++c) {
-        YK[c] = sumY - lookup( mean(depvar,c), moments );
-        //std::cout << "YK: " << YK[c] << ' ';
-      }
-      //std::cout << std::endl;
+//      // Y|Kc
```

Comment:
Do we want to delete this? Is this something you're still working on (I may have seen that chain elsewhere?)